models/census.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var mongoose = require('mongoose');
+
+var censusSchema = mongoose.Schema({
+    'type': String,
+    'activity_year': String,
+    'data': [{
```

Comment:
My concern is that the way that most of these are structured, you would end up storing all of the data for a single activity_year in one document within mongo. For something like census or the msaBranches, is it possible that you might run up against the 16M limit on documents?